tfgrid3/ubuntu24.04/microvm/zinit/ssh-init.yaml

```diff
@@ -0,0 +1,3 @@
+exec: bash /start.sh
```

Comment:
better to use ```exec: start.sh ``` directly to just use one process which is exec , not two which is zinit and bash

Reply:
- updated